docs/installation/docker-compose_install.md

```diff
@@ -0,0 +1,174 @@
+!!! Note
+    For test and development purposes we recommend to use the dockerized ODTP.
+    However this makes use of "Docker in Docker" to run the digital executions which can lead to security issues. More information [here](https://jpetazzo.github.io/2015/09/03/do-not-use-docker-in-docker-for-ci/).
+
+For the setup with docker compose the following is provided at [https://github.com/odtp-org/odtp](https://github.com/odtp-org/odtp)
+
+- `compose.yml` file that contains secrets as variables
+- `.env.dist.compose` file that contains all variables that need to be set for the `docker compose`
+
+The Installation  is done in the following steps: 
+
+[TOC]
+
+## Prerequisites
+
+In order to install ODTP with Docker compose you need [Docker compose](https://docs.docker.com/compose/install/) additionally to [Docker](https://www.docker.com/) and [git](https://git-scm.com/).
+
+## 1. Set up a folder configuration 
+
+Create a folder (we recommend you name it `odtp`) where ODTP will locate all services, and files needed.
+Create the following sub-folders: `mongodb`, `minio`, and `digital-twins`.
+The file system structure should be like this:
+
+```bash
+└── odtp
+    ├── mongodb
+    ├── minio
```

Comment:
Add a folder called: `odtp` under minio. This simplifies the process as the user doesn't need to log in `minio` and create the folder manually.